src/array_api_extra/_lib/_libraries.py

```diff
@@ -0,0 +1,69 @@
+"""Code specifying libraries array-api-extra interacts with."""
```

Comment:
This isn't quite right, e.g. ndonnx should work out of the box iff it is truly array API compliant (although `at` will additionally require it to be writeable and accepting None-sized bool masks)